src/esm_runscripts/filedicts.py

```diff
@@ -63,10 +67,68 @@ def cp(self) -> None:
     def ln(self) -> None:
         pass
 
-    def mv(self) -> None:
-        pass
-        
-        
+    def mv(self, source: str, target: str) -> None:
+        """
+        Moves (renames) the SimulationFile from it's location in ``source`` to
+        it's location in ``target``.
+
+        Parameters
+        ----------
+        source : str
+            One of ``"pool"``, ``"work"``, ``"exp_tree"``, "``run_tree``"
+        target : str
+            One of ``"pool"``, ``"work"``, ``"exp_tree"``, "``run_tree``"
+        """
+        if source not in self.locations:
+            user_error(
+                "Filedict Error",
+                "source is incorrectly defined, and needs to be in {self.locations}",
+            )
+        source_path, target_path = self._determine_names(source, target)
+        # Perform the movement:
+        try:
+            source_path.rename(target_path)
+            logger.success(f"Moved {source_path} --> {target_path}")
+        except Exception:  # Probably better to look for specific "breaking" things here
+            user_error(
```

Comment:
Same here. I think `IOError` would be better.

Reply:
OK, it has `IOError` now, but with a customized message.